src/main/kotlin/org/misarch/address/graphql/input/CreateVendorAddressInput.kt

```diff
@@ -0,0 +1,13 @@
+package org.misarch.address.graphql.input
+
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
+
+@GraphQLDescription("Input for the createVendorAddress mutation.")
+class CreateVendorAddressInput(
+    street1: String,
+    street2: String,
+    city: String,
+    postalCode: String,
+    country: String,
+    companyName: String,
```

Comment:
again, voluntary

Reply:
On second thought: Do you want to require that all vendors are companies?
So, a person cannot just sell something?

Reply:
we are not a platform
only we can sell sth
or what do you mean?

Reply:
Why do we have a Vendor address then?

Reply:
because return shippings need an address
and for some reason the invoice pdf also needs our address...